packages/common/tests/webrtc/helpers.test.ts

```diff
@@ -198,4 +198,71 @@ describe('Helpers browser functions', () => {
     })
 
   })
+
+  describe('Filter UDP(Defaults) Tests', () => {
+    it('Should not filter by default', () => {
+      const iceServers = [
+        {
+          "urls": [
+            "turn:turn.swire.io:443",
+            "turn:turn.swire.io:443?transport=tcp", 
+          ],
+          "credential": "secret",
+          "credentialType": "password",
+          "username": "1686088155:5b4b11d6-0824-4a66-b3f9-d5aabdf6ab8e"
+        }
+      ]
+
+      const filteredIceServers = filterIceServers(iceServers)
+
+      expect(filteredIceServers).toEqual(iceServers)
+
+    })
+
+    it('Should not filter if nothing is to filter', () => {
+      const iceServers = [
+        {
+          "urls": [
+            "turn:turn.swire.io:443?transport=tcp", 
+          ],
+          "credential": "secret",
+          "credentialType": "password",
+          "username": "1686088155:5b4b11d6-0824-4a66-b3f9-d5aabdf6ab8e"
+        }
+      ]
+
+      const filteredIceServers = filterIceServers(iceServers, {disableUDP: true})
+
+      expect(filteredIceServers).toEqual(iceServers)
+    })
+
+    it('Should filter only non transport=tcp URI', () => {
+
+      const iceServers = [
+        {
+          "urls": [
+            "turn:turn.swire.io:443",
+            "turn:turn.swire.io:443?transport=tcp", 
+          ],
+          "credential": "secret",
+          "credentialType": "password",
+          "username": "1686088155:5b4b11d6-0824-4a66-b3f9-d5aabdf6ab8e"
+        }
+      ]
+
+      const filteredIceServer = filterIceServers(iceServers, {disableUDP: true})
+
+      expect(filteredIceServer).toEqual([
+        {
+          "urls": [
+            "turn:turn.swire.io:443?transport=tcp", 
+          ],
+          "credential": "secret",
+          "credentialType": "password",
+          "username": "1686088155:5b4b11d6-0824-4a66-b3f9-d5aabdf6ab8e"
+        }
+      ])
+
+    })
+  })
```

Comment:
Maybe it's worth adding a test where the `urls` array contains a URL with `?transport=udp` (which should be treated as if the `transport` was omitted, defaulting to UDP). It doesn't change anything right now, but could be good for covering regressions in the future.